Sources/TuistAcceptanceTesting/TuistAcceptanceTestCase.swift

```diff
@@ -30,15 +34,21 @@ open class TuistAcceptanceTestCase: XCTestCase {
             LoggingSystem.bootstrap(AcceptanceTestCaseLogHandler.init)
         }
 
-        derivedDataDirectory = try! TemporaryDirectory(removeTreeOnDeinit: true)
-        fixtureTemporaryDirectory = try! TemporaryDirectory(removeTreeOnDeinit: true)
+        derivedDataDirectory = try TemporaryDirectory(removeTreeOnDeinit: true)
+        fixtureTemporaryDirectory = try TemporaryDirectory(removeTreeOnDeinit: true)
 
-        sourceRootPath = try! AbsolutePath(
+        sourceRootPath = try AbsolutePath(
             validating: ProcessInfo.processInfo.environment[
                 "TUIST_CONFIG_SRCROOT"
             ]!
         )
 
+        setenv(
+            "XDG_CACHE_HOME",
+            cacheDirectory.path.pathString,
+            1
+        )
```

Comment:
😅😅 We need to move away from this pattern at some point for the sake of parallelization.